testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -4030,3 +4031,81 @@ if(devices.real().deviceType === 'desktop') {
         });
     });
 }
+
+if(QUnit.urlParams['nojquery']) {
+    QUnit.module('ShadowDOM', {
+    }, () => {
+        QUnit.test('ShadowDom: drag item', function(assert) {
```

Comment:
We don't need to add "ShadowDom" prefix because it's already mentioned in the module name.

Reply:
ok